ReactCommon/butter/function.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <butter/butter.h>
+
+#ifdef defined(BUTTER_FUNCTION_OVERRIDE_INCLUDE) && \
+    defined(BUTTER_FUNCTION_OVERRIDE))
+
+#include <BUTTER_FUNCTION_OVERRIDE_INCLUDE>
```

Comment:
In the comment you said: `-DBUTTER_FUNCTION_OVERRIDE_INCLUDE=<functional/functor.h>`.
Then the code here should read: `#include BUTTER_FUNCTION_OVERRIDE_INCLUDE`
To avoid double `<<functional/functor.h>>`

Reply:
Sure. Thanks
Let's use
```
#include BUTTER_FUNCTION_OVERRIDE_INCLUDE
```